styles.css

```diff
@@ -1822,3 +1832,180 @@ i.background-msg-arrow {
     bottom: -195px;
   }
 }
+
+
+/* Auth Related, Needs refactoring */
+
+.modal-dialog {
+  width: 35%;
+}
+
+.modal-heading {
+  text-align: center;
+  margin-top: 15%;
```

Comment:
Change the values of margin and padding in pixels

Reply:
Ok